src/main/java/de/tum/in/www1/artemis/service/connectors/GitService.java

```diff
@@ -45,6 +43,7 @@
 
 import de.tum.in.www1.artemis.domain.*;
 import de.tum.in.www1.artemis.domain.File;
+import de.tum.in.www1.artemis.domain.Repository;
```

Comment:
This is necessary, otherwise it would get ambiguous since one of the jgit wildcard imports also includes a repository class - this is the main reason why many reject wildcard imports altogether